demo/color-swatch.js

```diff
@@ -1,7 +1,10 @@
-<link rel="import" href="../../polymer/polymer.html">
-<link rel="import" href="../d2l-colors.html">
-<dom-module id="d2l-demo-color-swatch">
-	<template strip-whitespace>
+import '../../@polymer/polymer/polymer-legacy.js';
+import '../d2l-colors.js';
+import { Polymer } from '../../@polymer/polymer/lib/legacy/polymer-fn.js';
+const $_documentContainer = document.createElement('template');
+
+$_documentContainer.innerHTML = `<dom-module id="d2l-demo-color-swatch">
+	<template strip-whitespace="">
```

Comment:
If you get rid of the `strip-whitespace` before the conversion, the html template will be put in the Polymer `_template` property. Although we miss the stripping of whitespace

Reply:
https://github.com/Polymer/polymer/issues/4731

Reply:
So I actually added this so that it would be consistent with the rest of our modularized components.

Reply:
Besides the different formatting of the modularized out (which ideally we'd have consistent), the other format results in some [spaces] rather than [tabs] which fails linting.  So this is partly to avoid those linting error s as well.

Reply:
Hmm. But wouldn't the script handle that conversion?

Reply:
It could.  Stacey and I spoke about this as an option, but ideally the output from the script is consistent anyway so we decided to not pursue that (yet)..

Reply:
Fair enough

Reply:
It seems like the modulizer adds 2 spaces instead of 4 for a tab, so we can update the script to try to get that consistent.  But yeah, we see style modules using the `innerHTML`, so we were thinking for consistency it's better to use that style in all files

Reply:
Thanks for pointing out - if it comes up more, I think we can definitely update the script to swap those spaces.  :)